Ugizwenayo-Divine/RICAINC-frontend
# Setting up the App and adding the Signup Component

### What does this PR do?
- It sets up the whole FrontEnd project
- It adds all the required features in order to create
### Description of the task to be completed
- Creating a Signup Component 
- Creating reducers
- Creating stores
- Creating middlewares
- Creating helpers
- Adding the styles to the component
- Setting up the entry point
- Creating actions
- Creating actionTypes
### How should this be manually tested?
- Clone the project
- write this command to install all the required dependencies
```
npm i
```
- write this command to start the project
```
npm run start
```
- make sure your backend server is running before starting the frontend project
- once you have started the project, the browser opens ```on localhost:3001``` , make sure you add /signup
